### PR TITLE
added TOC to reference attributes

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: reference/conanfile/attributes/binary_model.inc [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+- **unknown**: The type of the package is unknown.
+
+
+.. _conan_conanfile_properties_settings:
+
+settings
+--------
+
+List of strings with the first level settings (from ``settings.yml``) that the recipe
+need, because:
```
Comment (RubenRBS) [inline reference/conanfile/attributes/binary_model.inc:40]:
```suggestion
needs, because:
```

File: reference/conanfile/attributes/build.inc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+build_policy
+------------
+
+Controls when the current package is built during a ``conan install``.
+The allowed values are:
+    
+- ``"missing"``: Conan builds it from source if there is no binary available.
+- ``"never"``: This package cannot be built from sources, it is always created with
+  ``conan export-pkg``
+- ``None`` (default value): This package won't be build unless the policy is specified
```
Comment (RubenRBS) [inline reference/conanfile/attributes/build.inc:40]:
```suggestion
- ``None`` (default value): This package won't be built unless the policy is specified
```

File: reference/conanfile/attributes/folders.inc [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+no_copy_source
+--------------
+
+The attribute ``no_copy_source`` tells the recipe that the source code will not be copied from
+the ``source_folder`` to the ``build_folder``. This is mostly an optimization for packages
+with large source codebases or header-only, to avoid extra copies.
+
+
+If you activate it (``no_copy_source=True``), is **mandatory** that the source code must not be modified at all by
```
Comment (RubenRBS) [inline reference/conanfile/attributes/folders.inc:101]:
```suggestion
If you activate ``no_copy_source=True``, it is **mandatory** that the source code must not be modified at all by
```

File: reference/conanfile/attributes/sources.inc [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+.. _exports_sources_attribute:
+
+exports_sources
+---------------
+
+List or tuple of strings with file names or
+`fnmatch <https://docs.python.org/3/library/fnmatch.html>`_ patterns that should be exported
+and will be available to generate the package. Unlike the ``exports`` attribute, these files
+shouldn’t be used by the ``conanfile.py`` Python code, but to compile the library or generate
```
Comment (RubenRBS) [inline reference/conanfile/attributes/sources.inc:44]:
```suggestion
shouldn't be used by the ``conanfile.py`` Python code, but to compile the library or generate
```

File: reference/conanfile/methods/generate.rst [hunk 5]
```diff
@@ -113,3 +113,159 @@ And copying the shared libraries in Windows and OSX to the current build folder,
...
+  an error and can raise at any time, even if it doesn't raise yet.
+- It is not possible either to call any methods or any attempt to reuse code from the dependencies
+  via this mechanism.
+- This information does not exist in some recipe methods, only in those methods that evaluate
+  after the full dependency graph has been computed. It will not exist in ``configure()``, ``config_options``,
+  ``export()``, ``export_source()``, ``set_name()``, ``set_version()``, ``requirements()``,
+  ``build_requirements()``, ``system_requirements()``, ``source()``, ``init()``, ``layout()``.
+  Any attempt to use it in these methods can raise an error at any time.
+- At the moment, this information should only be used in ``generate()`` and ``validate()`` methods.
+  Any other use, please submit a Github issue.
```
Comment (RubenRBS) [inline reference/conanfile/methods/generate.rst:162]:
```suggestion
  For any other use, please submit a Github issue.
```

**Review by RubenRBS — APPROVED** [review]
👍

File: conf.py [hunk 6]
```diff
@@ -118,7 +118,7 @@
 # non-false value, then it is used:
 #today = ''
 # Else, today_fmt is used as the format for a strftime call.
-#today_fmt = '%B %d, %Y'
+#today_fmt = '%B %d, %Y'git
```
[the remark this answers is not in the document]
Comment (memsharded) [reply conf.py]: Fixed

**Review by czoido — APPROVED** [review]

**Pull request merged**